src/uct/api/v2/uct_v2.h

```diff
@@ -413,6 +417,18 @@ struct uct_ep_attr {
      * Remote sockaddr the endpoint is connected to.
      */
     struct sockaddr_storage remote_address;
+
+    /**
+     * Opaque value that reflects current position in tx completion queue.
```

Comment:
It's not clear how have a deal with this opaque value.  What happens in case of overflow? Is uint32_t max value reachable?

Reply:
uint32_t is reachable, but cannot be overflowed